### ISSUE TITLE
[BUG] Hitman Blood Money graphical and performance issues

### ISSUE THREAD

**Issue by ZT-1** [PCSX2/pcsx2#4251]
**Describe the bug**
Several graphical issues with opengl and d3d11. Performance becomes very slow at anything past 2x native.
![pcsx2_2021-02-24_13-03-07](https://user-images.githubusercontent.com/11493450/109047135-86e78a00-76a3-11eb-8305-85077c350356.jpg)
![pcsx2_2021-02-24_13-04-28](https://user-images.githubusercontent.com/11493450/109047138-89e27a80-76a3-11eb-9dc9-4017f2116c10.jpg)

Software mode doesn't have these graphical issues, but performance isn't great even with auto flush disabled. MTVU does seem to help though with d3d/opengl.
![pcsx2_2021-02-24_13-06-18](https://user-images.githubusercontent.com/11493450/109047364-c8783500-76a3-11eb-8534-f4a1592e75f8.jpg)
![pcsx2_2021-02-24_13-07-47](https://user-images.githubusercontent.com/11493450/109047369-cada8f00-76a3-11eb-8a6e-79a6af1f94c7.jpg)

**To Reproduce**
Just booting up.

**Expected behavior**
Should look more accurate like how it does in software mode.

**GS Settings**
![pcsx2_2021-02-24_13-28-52](https://user-images.githubusercontent.com/11493450/109048561-38d38600-76a5-11eb-9e7a-66b03fcfa334.png)

**Emulation Settings**
![pcsx2_2021-02-24_13-36-36](https://user-images.githubusercontent.com/11493450/109048685-5c96cc00-76a5-11eb-8e06-e95242b28ec5.png)

**System Info (please complete the following information):**
 - PCSX2 Revision: v1.7.0-dev-1009-gb088ee69c
 - OS: Windows 10
 - CPU: i7 4790
 - GPU: GTX 1060

**Logs and Dumps**
https://mega.nz/file/L0QABbYC#Es9VlSKSoU-s7Z4vSKf1rKMjRRsPO1owFFO048HnzfE

**Comment by tadanokojin** [comment]
Yeah I know about it. The game has a lot of nasty post-processing.
You'll be waiting a while on this one.

**Comment by DNS** [comment]
Still not working on PCSX2 Nightly - v1.7.4500
![Hitman - Blood Money_SLUS-21108_20230514142856](https://github.com/PCSX2/pcsx2/assets/37093/e3ff2ea8-7fe8-40cd-b3b1-13b8b4738617)

**Comment by MBeeBR** [comment]
that rectangle on the left seems to have something to do with it all 🤔

![unknown_2023 07 04-16 32](https://github.com/PCSX2/pcsx2/assets/136722551/afa3ace5-e757-461a-b6cf-563fd25a43d7)
![unknown_2023 06 14-07 43_1](https://github.com/PCSX2/pcsx2/assets/136722551/bf293fd2-1883-4f9b-ae9e-bd919d6d9b4e)

**Comment by RedDevilus** [comment]
The rectangle behaviour on top left, happens a lot if something is wrong on the texture cache side of things.

**Comment by MBeeBR** [comment]
in this case I think that small square is a filter, stretching it over the entire screen should solve the problem. and the little square makes the red spots bug disappear when they pass through it

**Comment by stenzek** [comment]
Such comments are not helpful. The game's doing a channel shuffle, reinterpreting render target data as a paletted/indexed texture, one page at a time. Hence why it only hits the top-left, because the other pages aren't in the texture cache.

You can't just "stretch it over the screen". It's no different from any other small sprite.

Such suggestions just fuel "devs are lazy" remarks, because those making them generally don't have any clue as to the complexity of what's going on.

**Comment by RedDevilus** [comment]
Seeing a square in top-left is also a common issue with games that have issues with the texture cache either in not being emulated properly or other root cause.

Either way here you see a listing of all the different types of shuffles: 

https://github.com/PCSX2/pcsx2/discussions/4109

**Comment by MBeeBR** [comment]
I'm not a programmer, I'm just a guy trying to help somehow solve this problem, each new version gives me the hope that everything will work perfectly 😌

**Comment by stenzek** [comment]
Then I'd kindly request to keep such comments away from the bug tracker. It's not helpful - if there was a simple solution, we would've done it already.

We *could* probably CRC hack it, but we're trying to get rid of those these days, and find general solutions, not add more.

**Comment by MBeeBR** [comment]
ok 😉

**Comment by witchkilla** [comment]
this is so sad. terrible

**Comment by kaidas1980** [comment]
Any News about a fix?
